<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiNetworkListItem</class>
 <widget class="QWidget" name="WifiNetworkListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>112</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="WifiIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>rsrc/wifi-icon.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="WifiStrength">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>20%</string>
   </property>
  </widget>
  <widget class="QLabel" name="WifiSSID">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SSID&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WifiHWAddr">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>15</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>HWAddr</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WifiChan">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>35</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Channel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LockIcon">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>rsrc/lock.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="WifiCrypto">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>WPA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BitrateIcon">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>55</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>N</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WifiFirstSeen">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>55</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>First Seen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WifiLastSeen">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>75</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Last seen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="backround">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-6</y>
     <width>841</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WifiHWAddr_name">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>15</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>HW Address: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WifiChan_name">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>35</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Channel: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WifiFirtstSeen_name">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>55</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>First seen: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="WifiLastSeen_name">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>75</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Last seen: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>backround</zorder>
  <zorder>WifiChan</zorder>
  <zorder>WifiFirstSeen</zorder>
  <zorder>WifiHWAddr</zorder>
  <zorder>WifiIcon</zorder>
  <zorder>WifiLastSeen</zorder>
  <zorder>WifiSSID</zorder>
  <zorder>WifiStrength</zorder>
  <zorder>BitrateIcon</zorder>
  <zorder>LockIcon</zorder>
  <zorder>WifiCrypto</zorder>
  <zorder>WifiHWAddr_name</zorder>
  <zorder>WifiChan_name</zorder>
  <zorder>WifiFirtstSeen_name</zorder>
  <zorder>WifiLastSeen_name</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
